net/netdev/netdev_ioctl.c

```diff
@@ -1072,7 +1072,7 @@ static int netdev_ifr_ioctl(FAR struct socket *psock, int cmd,
           if (dev && dev->d_ioctl)
             {
               struct can_ioctl_data_s *can_bitrate_data =
-                &req->ifr_ifru.ifru_can_bitrate;
+                &req->ifr_ifru.ifru_can_data;
```

Comment:
let's revert these change in "arch/stm32h7: Add SocketCAN FDCAN driver" directly.

Reply:
I think this is done to be consistent with line 1093 of the added code